server/game/gameSystems/GameSystemLibrary.ts

```diff
@@ -146,12 +146,15 @@ export function lookAt<TContext extends AbilityContext = AbilityContext>(propert
 export function moveCard<TContext extends AbilityContext = AbilityContext>(propertyFactory: PropsFactory<IMoveCardProperties, TContext>): CardTargetSystem<TContext> {
     return new MoveCardSystem<TContext>(propertyFactory);
 }
-// /**
-//  * default resetOnCancel = false
-//  */
-// export function playCard(propertyFactory: PropsFactory<PlayCardProperties> = {}): GameSystem {
-//     return new PlayCardAction(propertyFactory);
-// }
+/**
+ * default resetOnCancel = false
+ */
+export function playCardFromHand<TContext extends AbilityContext = AbilityContext>(propertyFactory: PropsFactory<Omit<IPlayCardProperties, 'playType' | 'optional'>, TContext> = {}): PlayCardSystem<TContext> {
```

Comment:
What's the reason to block out `optional`?

Reply:
Playing from hand will always be optional because it's a hidden zone. Right?

Reply:
Oh yeah true. I guess the only difference is that we potentially wouldn't need to show the extra explainer text saying why the play is optional when they might not expect it to be

Reply:
Do you mean that we won't want to show the explainer text when the card text says "may"?

Reply:
Ah hey sorry missed your reply here - I realized there isn't actually a case where this would apply so nvm.